cmd/gendocs/gendocs.go

```diff
@@ -165,6 +165,9 @@ rclone.org website.`,
 					if name != "rclone.md" {
 						return fmt.Errorf("internal error: failed to find cut points: startCut = %d, endCut = %d", startCut, endCut)
 					}
+					if endCut >= 0 {
+						doc = doc[:endCut] + "### See Also" + doc[endCut+12:]
```

Comment:
I think the "SEE ALSO" got cribbed from man pages but it markdown headings are so much better :-)